deployment/ansible/roles/install_config_nginx/templates/nginx.conf.j2

```diff
@@ -0,0 +1,14 @@
+# roles/nginx/templates/nginx.conf.j2
+server {
+    listen 80;
+
+    server_name {{ ansible_fqdn }};
```

Comment:
not sure what will be in your case in this var, but you can use:

server_name _;  accepts all hostnames